HW02/Arthur_Bohan/arthur_bohan_hw_02_t_02.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# create regex
+IP_ADDRESS = /^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/.freeze
+DATE = /\[.*\]/.freeze
+PATH = /POST\s\S+/.freeze
+
+# read txt file as array
+lines = File.readlines('input_hw_02_t_02.txt', chomp: true)
+
+# formatd lines information
+def formated_log(log)
+  log.select { |line| line.match(DATE) }.map do |line|
```

Comment:
Try to use more strict regexp, at least include `IP_ADDRESS`, `DATE` and `PATH`.` LINE_REGEXP = /.../`
```suggestion
  log.select { |line| line.match(LINE_REGEXP) }.map do |line|
```